### DISCUSSION TITLE
Tui renderer error

### DISCUSSION THREAD

**Discussion by Wanpaku** [tracel-ai/burn#1862] [Q&A] [answered]
I've tried to run yours  my-first-burn-model example and this error appeared:

ERROR burn_train::learner::log: PANIC => panicked at /.cargo/registry/src/index.crates.io-6f17d22bba15001f/burn-train-0.13.2/src/renderer/tui/renderer.rs:85:27:
called `Result::unwrap()` on an `Err` value: Os { code: 6, kind: Uncategorized, message: "No such device or address" }  

How to fix it?

**Comment by laggui** [comment]
What are you trying to run the executable from? If you're trying to run from your IDE (e.g., vscode) it looks like you might encounter a similar error.

**Reply by Wanpaku** [reply, marked as answer]
First time I run it through Kate text editor and the error appeared.
Now I run it through Vim and all work as expected.

**Reply by laggui** [reply]
Ok so it's probably an issue with crossterm and your IDE. Glad you could get it to work on Vim!
